journals/apps/core/models.py

```diff
@@ -34,6 +39,54 @@ def get_full_name(self):
     def can_access_admin(self):
         return self.has_perm('wagtailadmin.access_admin')
 
+    @classmethod
+    def get_user_by_username(cls, username):
+        """
+        Returns the user object with given username
+        if that username is valid else returns None
+        """
+        try:
+            user = cls.objects.get(username=username)
+        except cls.DoesNotExist:
+            user = None
+        return user
+
+    @classmethod
+    def does_username_exit(cls, username):
```

Comment:
I think `get_user_by_username` method can be used to determine if user exists or not instead of defining a separate method.